bedrock/firefox/templates/firefox/base-pebbles.html

```diff
@@ -4,18 +4,42 @@
 
 {% extends "base-pebbles.html" %}
 
+{% set quantum = switch('firefox-57-release') %}
+
 {% block page_title_prefix %}{% endblock %}
 {% block page_title_suffix %}{% endblock %}
+
 {% block page_image %}
-{% if switch('firefox-57-release') %}
+{% if quantum %}
   {{ static('img/firefox/template/page-image-quantum.png') }}
 {% else %}
   {{ static('img/firefox/template/page-image.png') }}
 {% endif %}
 {% endblock %}
-{% block page_favicon %}{{ static('img/firefox/favicon.ico') }}{% endblock %}
-{% block page_favicon_large %}{{ static('img/firefox/favicon-196.png') }}{% endblock %}
-{% block page_ios_icon %}{{ static('img/firefox/ios-icon-180.png') }}{% endblock %}
+
+{% block page_favicon %}
+{% if quantum %}
```

Comment:
Hmm, it seems this indentation is adding an extra space before and after the `href`? Should still be ok though?

Reply:
Pretty sure you can fix this with the no-whitespace controls:

`{%- if quantum -%}`

Reply:
Ah, yes! thanks

Reply:
Yeah whitespace *should* be ignored in HTML but it does look much nicer without it.